app/serializers/email_serializer.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class EmailSerializer < ActiveModel::Serializer
+  attribute :email_address
+  attribute :effective_at
+
+  def id
+    nil
+  end
+
+  # Returns the email address nested in the given object.
+  #
+  # @return [String] Email address.  Sample `object.email_address`:
+  #   {
+  #     "effective_date" => "2018-02-27T14:41:32.283Z",
+  #     "value" => "test2@test1.net"
+  #   }
+  #
+  def email_address
```

Comment:
I know I already made a naming suggestion here, but I have a nitpicky consistency suggestion - if we're already using `/v0/profile/email`, maybe we want to settle on the use of `email` (both internally and externally) instead of `email_address`. Or, we could set up `/v0/profile/email_address`.

Reply:
@jkassemi - Yeah, good call.  Agreed.  I lean towards setting them all to `email`...short and sweet.

I'll make the appropriate changes!

cc: @bshyong @ncksllvn